sdk/eventgrid/azure-eventgrid/azure/eventgrid/_shared/mixins.py

```diff
@@ -0,0 +1,58 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint:disable=protected-access
+
+class DictMixin(object):
```

Comment:
@lmazuel I wish we had like, "python_sdk_utils" that contained generic helpers like this, since I'm 100% sure this exists in multiple of our libs.

(I realize this statement sounds like airy musings and not something actionable, so let me rephrase: "is this an insane proposition to you?")

Reply:
might be something that can move to azure core